cryptoki/tests/basic.rs

```diff
@@ -1326,7 +1348,14 @@ fn aes_gcm_with_aad() -> TestResult {
         Attribute::Encrypt(true),
     ];
     let key_handle = session.create_object(&template)?;
-    let mechanism = Mechanism::AesGcm(GcmParams::new(&mut iv, &aad, 96.into()));
+    let gcm_params = match GcmParams::new(&mut iv, &aad, 96.into()) {
+        Ok(params) => params,
+        Err(e) => { 
+            println!("{e}");
+            return Err(e.into());
+        },
+    };
```

Comment:
```suggestion
    let gcm_params = match GcmParams::new(&mut iv, &aad, 96.into())?;
```

`TestResult` should take care of printing and panicking when this fails.